sample/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdk 30
+
+    defaultConfig {
+        applicationId "co.nimblehq.sample"
+        minSdk 21
+        targetSdk 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+    }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = '1.8'
+    }
+    buildFeatures {
+        viewBinding true
+    }
+}
+
+dependencies {
+    implementation(
+            project(':survey-sdk')
```

Comment:
I think we only use 4 spaces for indent? 🤔

Reply:
@hoangnguyen92dn I'm not sure why AS not allow me to use 4 spaces in this case, event I set the next indent is 4 spaces

<img width="237" alt="Screen Shot 2021-09-10 at 5 17 53 PM" src="https://user-images.githubusercontent.com/88996251/132839065-e095ecae-0a99-4b29-88a4-7e5b28c25165.png">

Reply:
@doannimble Did you update on the `Groovy` style?
![image](https://user-images.githubusercontent.com/6950766/132840837-ba092987-a0b3-42a3-ae02-8a6e4855c237.png)

Reply:
great @hoangnguyen92dn , setting up Groovy resolve this. Thanks!